sos/policies/redhat.py

```diff
@@ -281,6 +287,61 @@ def check(cls):
                         return True
         return False
 
+    def get_upload_url(self):
+        if self.case_id and not self.commons['cmdlineopts'].upload_url:
+            # in this case we want to default to using the RHCP for uploads so
+            # clear the default FTP user so that sos users will be prompted
+            # for an RHCP username
+            self.upload_user = None
+            # this string is only used in pre_work() prompts, so this does not
+            # need to be the actual RHCP url
+            return "Red Hat Customer Portal"
+        return RH_FTP_HOST or self.commons['cmdlineopts'].upload_url
```

Comment:
Again, should not `cmdlineopts` option (if set) have preference to `RH_FTP_HOST`?